packages/manager/apps/telecom/src/components/notification/translations/Messages_fr_FR.json

```diff
@@ -6,13 +6,15 @@
   "components_notification_noNotification": "Il n'y a aucune alerte enregistrée.",
   "components_notification_frequency": "Fréquence d'envoi",
   "components_notification_contact": "Contact à notifier en cas d'alertes",
-  "components_notification_account": "Compte:",
+  "components_notification_account": "Compte",
   "components_notification_confirm": "Êtes-vous sûr(e) de vouloir supprimer la notification sur le service {{service}} pour {{contact}} ?",
   "components_notification_add": "Ajouter une alerte",
   "components_notification_mail": "E-mail",
   "components_notification_sms": "SMS",
   "components_notification_once": "Une fois",
   "components_notification_5m": "Toutes les 5 minutes",
   "components_notification_1h": "Toutes les heures",
-  "components_notification_6h": "Toutes les 6 heures"
-}
\ No newline at end of file
+  "components_notification_6h": "Toutes les 6 heures",
+  "components_notification_sms_detail": "Pour le compte {{smsAccount}}, il vous reste actuellement {{creditsLeft}} € de crédit.",
```

Comment:
We should retrieve dynamically customer currency code. Here it is ok because it is only available in France and Spain but what if offer is launched in Poland ?

Reply:
Yes we should, but it need to update API to retrieve currency from the credits left.

Reply:
It's a misunderstood from me, credits left are not using currency. So I removed it from json file